compiler/modules/types/mapping.bal

```diff
@@ -1,6 +1,8 @@
 // Implementation specific to basic type list.
 
-public type Field [string, MemberSemType];
+public type Field [string, SemType];
```

Comment:
Why do we need both Field and MemberField here?

Reply:
Well, in the `defineListTypeWrapped()` we take a `SemType[]` and convert it to a `CellSemType[]`.
Similarly in the `defineMappingTypeWrapped()` we take a `Field[]` and convert it to a `CellField[]`. So we need two type definitions to represent `[string, SemType]`(input) and `[string, CellSemType]`(after wrapping) The current occurrence of `CellField` is only `mapping.bal`.